src/DotVVM.Framework/Controls/GridViewTextColumn.cs

```diff
@@ -40,18 +40,6 @@ public ICommandBinding ChangedBinding
             DotvvmProperty.Register<ICommandBinding, GridViewTextColumn>(t => t.ChangedBinding, null);
 
         /// <summary>
-        /// Gets or sets the type of value being formatted - Number or DateTime.
-        /// </summary>
-        [MarkupOptions(AllowBinding = false)]
-        public FormatValueType ValueType
-        {
-            get { return (FormatValueType)GetValue(ValueTypeProperty); }
-            set { SetValue(ValueTypeProperty, value); }
-        }
-        public static readonly DotvvmProperty ValueTypeProperty =
-            DotvvmProperty.Register<FormatValueType, GridViewTextColumn>(t => t.ValueType);
```

Comment:
This property was also removed in version 2.0 and it's also used in the dothtml markup, which does not give you any way to remove all obsolete usages and will not report an error when the property is misused. I think that we should also add this one back.

Reply:
I am not sure here. Will it break anything if we remove it? It will, in this case, because bindable objects do not support HTML attributes. But in other cases it will just render an unwanted HTML attribute. @quigamdev What do you think?